sorting-verify/insertion-sort.js

```diff
@@ -1,31 +1,24 @@
-function leftMoving(newArr, n){
-   if(newArr[n]<newArr[n-1]){
-     var temp = newArr[n-1];
-     newArr[n-1] = newArr[n];
-     newArr[n] = temp;
-
-     do {
-       n = n-1;
-       leftMoving(newArr, n);
-     } while (n>1);
+function swap(arr, m, n) {
+  var temp = arr[m];
+  arr[m] = arr[n];
+  arr[n] = temp;
+}
 
```

Comment:
Do not need to indicate the relations between m and n just yet. It is included when calling the function.